dask/array/core.py

```diff
@@ -1047,6 +1047,19 @@ def _set_chunks(self, chunks):
     def __len__(self):
         return sum(self.chunks[0])
 
+    def __array_ufunc__(self, numpy_ufunc, method, *inputs, **kwargs):
+        import pdb; pdb.set_trace()
+        from . import ufunc
+        da_ufunc = getattr(ufunc, numpy_ufunc.__name__)
```

Comment:
We're now returning `NotImplemented` when we don't know how to handle the inputs.